### PR TITLE
[TGBOT-288] Create parent pom and child modules

### PR DESCRIPTION
**Pull request by yurasava**
https://schneidervs.atlassian.net/jira/software/projects/TGBOT/boards/2?selectedIssue=TGBOT-288

### CODE REVIEW

**Review by Sav4ykVladzislau — COMMENTED** [review]
lgfm

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 HELP.md
-src/main/resources/application.properties
+*.properties
```
Comment (max-java) [inline .gitignore]: properties files should be in the repository
if you have sensitive data in there - that should be encrypted or replaces with placeholders

File: pom.xml [hunk 2]
```diff
@@ -1,75 +1,94 @@
...
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>3.0.2</version>
```
Comment (max-java) [inline pom.xml]: declare versions as properties, and use variables here

File: pom.xml [hunk 3]
```diff
@@ -1,75 +1,94 @@
...
+                <artifactId>spring-boot-starter-thymeleaf</artifactId>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-web</artifactId>
+            </dependency>
+            <dependency>
+                <groupId>org.thymeleaf.extras</groupId>
+                <artifactId>thymeleaf-extras-springsecurity6</artifactId>
+            </dependency>
```
Comment (max-java) [inline pom.xml]: you don't need this in dependency management, as it manages by spring bom

**Review by max-java — CHANGES_REQUESTED** [review]
minor comments, otherwise lgfm

**Review by Sav4ykVladzislau — APPROVED** [review]

**Review by schneidervs — APPROVED** [review]

**Review by Artem-Metlushko — APPROVED** [review]

**Review by Artem-Metlushko — APPROVED** [review]

**Review by AlexanderYarokhovich — APPROVED** [review]

**Review by max-java — APPROVED** [review]
lgfm

**Pull request merged**